frontend/src/components/alert/alert.tsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect } from 'react';
+import ReactDOM from 'react-dom';
+import {
+  AiOutlineCloseCircle,
+  AiOutlineCheckCircle,
+  AiOutlineInfoCircle,
+} from 'react-icons/ai';
+import { AlertPorps } from '../../interfaces/types';
+import { theme } from '../../theme';
+import {
+  AlertBox,
+  Close,
+  Message,
+  Title,
+  ContentContainer,
+  CloseContainer,
+} from './alert-styles';
+
+const alertPortal = document.getElementById('alert-root');
+
+const AlertContainer = ({ type, title, message, onClose }: AlertPorps) => {
```

Comment:
I see that this is actually a portal ...so maybe you can name it more precise to be clear right away - AlertPortal or something like that? You can rename alertPortal to be alertPortalRoot or similar? So when another dev reads your code in formValidation and come across AlertPortal its going to be more clear for him that component is acctually a portal being attached outside of root div then AlertContainer